smoke-tests-otel-starter/spring-boot-3/build.gradle.kts

```
@@ -66,6 +66,9 @@ graalvmNative {
     // Workaround for https://github.com/junit-team/junit5/issues/3405
     buildArgs.add("--initialize-at-build-time=org.junit.platform.launcher.core.LauncherConfig")
     buildArgs.add("--initialize-at-build-time=org.junit.jupiter.engine.config.InstantiatingConfigurationParameterConverter")
+
+    // enable JFR - see https://www.graalvm.org/22.0/reference-manual/native-image/JFR/
```

This documentation if for an old GraalVM version.

The last one:
```suggestion
    // enable JFR - see https://www.graalvm.org/latest/reference-manual/native-image/debugging-and-diagnostics/JFR/
```